aws/table_aws_guardduty_detector.go

```diff
@@ -36,6 +36,13 @@ func tableAwsGuardDutyDetector(_ context.Context) *plugin.Table {
 				Type:        proto.ColumnType_STRING,
 				Transform:   transform.FromField("DetectorID"),
 			},
+			{
+				Name:        "arn",
+				Description: "The Amazon Resource Name (ARN) specifying the detector.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getAwsGuardDutyDetectorAkas,
```

Comment:
Change the function name
```suggestion
				Hydrate:     getGuardDutyDetectorARN,
```